crates/biome_css_semantic/src/semantic_model/model.rs

```diff
@@ -135,6 +139,27 @@ pub struct Selector {
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Default)]
 pub struct Specificity(pub u32, pub u32, pub u32);
 
+impl std::ops::Add for Specificity {
```

Comment:
It's worth adding a comment that explains how the `Add` is implemented for `Specificity` , maybe with a link to the spec

Reply:
Add comment at https://github.com/biomejs/biome/pull/4097/commits/ffec35e4631171563478b3a7a3c68039f1d0e5ec